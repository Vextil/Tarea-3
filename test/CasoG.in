InsertarCP t1 95 t2 95 e1 95 e2 95 g 95 s 95 t1 99 t2 5 a 95 b 95 c 95 d 95 e 95 f 95 g 95 FIN
ElementosCP 95 1
ExtraerCP
ExtraerCP
PartirLista 1 2
ImprimirLista 2
CopiaLista 1 3
RemoverDeLista 3 6
ImprimirLista 3
CrearBalanceado 2 1
Izquierdo 1 2
Derecho 1 3
AlturaCantidadBinario 2
AlturaCantidadBinario 1
Encolar 1
Encolar 2
Desencolar
AgregarProcesos pr30 100 pr20 200 pr80 100 pr50 200 pr60 150 pr10 200 pr15 200 FIN
EliminarProceso pr10
MuyConsumidores 100
ImprimirProcesos
ListarProcesos
CorrerProcesos pr30 100 pr20 200 pr80 100 pr50 200 pr60 150 pr10 200 pr15 200 FIN
PedirRecursos pr30 95 pr20 5 pr20 95 pr60 98 pr80 5 FIN
SolicitanOMuyConsumidores 100 95
Fin
