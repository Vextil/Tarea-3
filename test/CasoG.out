1>OK
2>OK
3>t2 - 5
4>t1 - 95
5>OK
6>Ordenada. a b c d e f g 
7>OK
8>s
9>NO ordenada. t1 t2 e1 e2 g 
10>OK
11>OK
12>OK
13>Altura: 2; Cantidad: 3
14>Altura: 3; Cantidad: 7
15>OK
16>OK
17>
   (0,g)
  (0,f)
   (0,e)
 (0,d)
   (0,c)
  (0,b)
   (0,a)

18>Hay 8 procesos.
19>La memoria que se libera es 200.
20>pr15 pr20 pr50 pr60 
21>
(150,pr60) 
(200,pr15) (200,pr50) 
(200,pr20) (100,pr80) 
(100,pr30) 
(100,boot) 
22>boot pr15 pr20 pr30 pr50 pr60 pr80 
23>OK
24>OK
25>pr10 pr15 pr20 pr30 pr50 pr60 
26>Fin
